src/index.js

```diff
@@ -22,6 +22,7 @@ import './core/MREntity';
 import './core/MRHand';
 import './core/MRSystem';
 import './core/MRTextEntity';
+import './core/MRHyperlink.js';
```

Comment:
are we sure we want this directly in core and not in entities - not too hardlined here but am leaning more towards in entities

Reply:
hmm I was thinking it might be one of those elements that gets extended but it was a coin toss. I'll move it.